test/usm/vector_add_auto_zero_copy/Makefile

```diff
@@ -0,0 +1,19 @@
+include ../../Makefile.defs
+
+TESTNAME     = vector_add_auto_zero_copy
+TESTSRC_MAIN = vector_add_auto_zero_copy.cpp
+TESTSRC_AUX  =
+TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
+
+CLANG        = clang++
+OMP_BIN      = $(AOMP)/bin/$(CLANG)
+CC           = $(OMP_BIN) $(VERBOSE)
+#-ccc-print-phases
+#"-\#\#\#"
+
+HSA_XNACK ?= 1
+
+include ../Makefile.rules
+
+run:
```

Comment:
Please use the `RUNCMD`  and `RUNENV` variables instead. The `run` rule is then automatically "generated".
Also, what is actually being tested here? Aren't we missing some `FileCheck` statements or other means to check that the runtime does the right thing?

```
RUNENV= HSA_XNACK=${HSA_XNACK} OMPX_APU_MAPS=1
```

Reply:
I added the RUNENV bit but left both tests without filecheck machinery. These are for demonstration purposes and not for compiler regression testing.